packages/cli/test/commands/build.test.ts

```diff
@@ -9,9 +9,9 @@ import { IConfig } from '@oclif/config'
 describe('build', () => {
   describe('Build class', () => {
     beforeEach(() => {
-      replace(configService, 'compileProject', fake.resolves({}))
-      replace(projectChecker, 'checkCurrentDirIsABoosterProject', fake.resolves({}))
-      replace(projectChecker, 'checkCurrentDirBoosterVersion', fake.resolves({}))
+      replace(configService, 'compileProject', fake())
+      replace(projectChecker, 'checkCurrentDirIsABoosterProject', fake())
+      replace(projectChecker, 'checkCurrentDirBoosterVersion', fake())
```

Comment:
I guess this change is OK-ish if the tests still pass, but notice that the logic is not the same, the old version is resolving a promise and returning an empty object, while the fake with no params does nothing AFAIK

Reply:
Actually this was compiling for no reason, because those methods return `void` while this resolved with an empty object. These tests passed because we weren't checking the result 😅